src/login/styles/_inputs.scss

```diff
@@ -122,6 +124,33 @@ input:focus {
   letter-spacing: 0.025rem;
 }
 
+.label-helpblock-container {
```

Comment:
I know most of this naming is legacy code, but as we are updating it could we rename these to be more descriptive? (i.e. is the helpBlock involved in this styling is it not just the label for the input?)